### PR TITLE
Set default third-party install path and modify how-to-build.md

### PR DESCRIPTION
**Pull request by laura-ding**

### CODE REVIEW

**Comment by laura-ding** [comment]
jenkins go

File: CMakeLists.txt [hunk 1]
```diff
@@ -171,6 +171,12 @@ if(NOT ${NEBULA_GPERF_BIN_DIR} STREQUAL "")
     list(INSERT CMAKE_PROGRAM_PATH 0 ${NEBULA_GPERF_BIN_DIR})
 endif()
 
+# When NEBULA_THIRDPARTY_ROOT is null, set default value is /opt/nebula/third-party
```
Comment (Amber1990Zhang) [inline CMakeLists.txt]: set default value as /opt/nebula/third-party

File: CMakeLists.txt [hunk 2]
```diff
@@ -171,6 +171,12 @@ if(NOT ${NEBULA_GPERF_BIN_DIR} STREQUAL "")
...
+if("${NEBULA_THIRDPARTY_ROOT}" STREQUAL "")
+    SET(NEBULA_THIRDPARTY_ROOT "/opt/nebula/third-party")
+    message(STATUS "Auto set NEBULA_THIRDPARTY_ROOT: " ${NEBULA_THIRDPARTY_ROOT})
```
Comment (Amber1990Zhang) [inline CMakeLists.txt]: STATUS "Set NEBULA_THIRDPARTY_ROOT automatically: "

File: CMakeLists.txt [hunk 3]
```diff
@@ -451,11 +453,15 @@ if (NOT SKIP_JAVA_CLIENT)
         message(FATAL_ERROR "Have not thrift-1.0-SNAPSHOT.jar")
```
Comment (Amber1990Zhang) [inline CMakeLists.txt]: "Doesn't have thrift-1.0-SNAPSHOT.jar"

File: docs/how-to-build.md [hunk 4]
```diff
@@ -40,11 +40,12 @@ The project itself includes the source code of several 3rd-party libraries, whic
...
+[Install guide](https://github.com/vesoft-inc/nebula-3rdparty/blob/master/README.md)
 
 
 ### How to build
   - 1. create build dir : `cd nebula && mkdir build && cd build`
-  - 2. generate makefile : `cmake ..`
+  - 2. generate makefile : `cmake -DNEBULA_THIRDPARTY_ROOT=${3rd-party_install_root} ..`
   - 3. make : `make or make -j${threadnum}`
   - 4. install: `make install`
   - notes : the default install dir is **/usr/local/**, if you want to change the dir, on step 2, your command can be `cmake -DCMAKE_INSTALL_PREFIX=$your_nebula_install_dir ..`
```
Comment (Amber1990Zhang) [inline docs/how-to-build.md]: the default installation dir is

Comment (laura-ding) [reply docs/how-to-build.md]: Thanks for your review @Amber1990Zhang

**Pull request merged**